docs/application-connector/docs/004-architecture-remote-environment.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
arch doc with no diagram and no real explanation of the architecture
- where is the RE coming from? 
- what is creating RE and EM Crs?
- as a result of such RE creation, what is happening in the system? 

there is even nothing about service classes

Reply:
The file will be removed and RE description will be in the general architecture doc.